src/context/flashmessage.tsx

```diff
@@ -0,0 +1,50 @@
+import { FunctionComponent, createContext, useState } from 'react'
+
+interface FlashMessageType {
+  title?: string
+  body: string
+  variant?: string
+  delay?: number
+}
+interface FlashContextType {
+  flashs: FlashMessageType[]
+  setFlashs: (flashs: FlashMessageType[]) => void
+  addFlash: (flash: FlashMessageType) => void
+}
+
+const flashDefault: FlashMessageType[] = []
+const flashsContextDefault = {
+  flashs: flashDefault,
+  setFlashs: () => {},
+  addFlash: () => {}
+}
+
+const flashsEqual = (flash1: FlashMessageType, flash2: FlashMessageType): boolean => (
+  JSON.stringify(flash1) === JSON.stringify(flash2)
+)
+
+const containsFlash = (flashs: FlashMessageType[], flash: FlashMessageType): boolean => (
+  flashs.map(
+    e => flashsEqual(e, flash)
+  ).reduce<boolean>(
+    (acc, cur) => acc || cur, false
+  )
+)
+
+const FlashContext = createContext<FlashContextType>(flashsContextDefault)
+
+const FlashContextProvider: FunctionComponent = ({ children }) => {
```

Comment:
Let's call it FlashMessageContextProvider

Reply:
fixed